command/secrets/init/helper.go

```diff
@@ -0,0 +1,71 @@
+package init
+
+import (
+	"github.com/0xPolygon/polygon-edge/crypto"
+	"github.com/0xPolygon/polygon-edge/network"
+	"github.com/0xPolygon/polygon-edge/secrets"
+	"github.com/0xPolygon/polygon-edge/types"
+	"github.com/libp2p/go-libp2p-core/peer"
+)
+
+// loadValidatorAddress loads ECDSA key by SecretsManager and returns validator address
+func loadValidatorAddress(secretsManager secrets.SecretsManager) (types.Address, error) {
+	if !secretsManager.HasSecret(secrets.ValidatorKey) {
+		return types.ZeroAddress, nil
```

Comment:
Why do you not even return an error here?

Reply:
This function is called only when generating command result. It's ok whether key doesn't exist because this value doesn't show in result.

Reply:
@zivkovicmilos ping